src/datasources/products/components/ProductsQueryEditor.tsx

```diff
@@ -144,9 +144,9 @@ export function ProductsQueryEditor({ query, onChange, onRunQuery, datasource }:
 }
 
 const tooltips = {
-  properties: "Select the properties fields to query",
-  recordCount: "Enter the number of records to query",
-  orderBy: "Select the field to order the results by",
-  descending: "Select to order the results in descending order",
-  queryBy: 'Enter the query to filter the results',
+  properties: "Specifies the fields to be queried from a product.",
+  recordCount: "Specifies the maximum number of products to return.",
+  orderBy: "Specifies the field to order the queried products by.",
+  descending: "Specifies whether to return the products in descending order.",
+  queryBy: 'Specifies the query to filter products. This is an optional field.',
```

Comment:
```suggestion
  queryBy: 'Specifies the filters to be applied on the products. This is an optional field.',
```

I would like to not use the "query" here, instead we could refer to the action of "filtering" which the user performs. @sudharsan-ni  any thoughts on this?

Reply:
Query is a pretty standard term in Grafana. As you can see, the tab in the Grafana panel editor is in itself called `Query`. For as labels, we can use `Query`... we already use `Query type`, 'Query by`, etc. but in the help text, we can use filter, filtering etc to refer to the action of filtering.